docs/dev/README.md

```diff
@@ -246,6 +246,14 @@ added to the profile via NVTX ranges.  See
 [the NVTX profiling guide](nvtx_profiling.md) for additional information on
 setting up the build for profiling and adding NVTX ranges.
 
+Note: Nsight Systems can be downloaded from its
+[product page](https://developer.nvidia.com/nsight-systems) directly.
+It is also installed as part of the CUDA Toolkit. To launch the GUI, run the following
+executable, depending on your CUDA Toolkit version:
+* `/usr/local/cuda-10.1/nsight-systems-2019.3.7.5/Host-x86_64/nsight-sys`
+* `/usr/local/cuda-10.2/nsight-systems-2019.5.2/host-linux-x64/nsight-sys`
+* `/usr/local/cuda-11.0/nsight-systems-2020.3.2/host-linux-x64/nsight-sys`
+
```

Comment:
with CUDA 11.2 I have on Ubuntu 18.4 I have:
```bash
$ which nsight-sys 
/usr/local/cuda-11.2/bin/nsight-sys
```

However, when I call it I get 
```
$ /usr/local/cuda-11.2/bin/nsight-sys
Error: Nsight Systems 2020.4.3 hasn't been installed with CUDA Toolkit 11.2
```

I wonder how much we should advertise this in the doc or need to add additional [caveats](https://forums.developer.nvidia.com/t/unable-to-install-nsys-nsight-system-cli/160704).

Reply:
Right, the wrapper script doesn't work, that's why I listed the executables. They also changed the name to `nsys-ui` for cuda 11.2. Like I said, it's kinda a mess.

Reply:
ack, nsys-ui in my install simply calls nsight-sys script in the same dir internally
```bash
$ cat /usr/local/cuda-11.2/bin/nsys-ui
#!/bin/bash
DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
exec "$DIR/nsight-sys"
```
It turns out that I actually don't have /usr/local/cuda-11.*/nsight-systems dir in my install
```
~$ find /usr/local/cuda-11.2/ -name nsight\*
/usr/local/cuda-11.2/bin/nsight-sys
/usr/local/cuda-11.2/bin/nsight_ee_plugins_manage.sh
/usr/local/cuda-11.2/nsightee_plugins
```

Reply:
That's strange. It should have been installed as part of the CUDA Toolkit: https://docs.nvidia.com/nsight-systems/InstallationGuide/index.html

Reply:
I strongly suggest encouraging people to always use the latest Nsight Systems rather than using what is bundled in the toolkit. You can miss out on a lot of important features and bug fixes by using what's in the toolkit. It's easy to download via `apt` or from the download site

Reply:
Good point. Changed the wording to recommend direct download.